<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1586</width>
    <height>1011</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>1220</width>
    <height>1010</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rendering in Novat style</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../../../../../../../../../images/RSN.ico</normaloff>../../../../../../../../../../../../../../images/RSN.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="mainWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1201</width>
      <height>781</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>740</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="reloadButton">
       <property name="text">
        <string>Reload</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="multiScaleGroupeBox">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>10</y>
      <width>361</width>
      <height>721</height>
     </rect>
    </property>
    <property name="title">
     <string>Multi-Scale</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>361</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="settingsLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
     </layout>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>770</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>810</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="informationTitle">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Current Shader :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="drawMode">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="drawModeSlider">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>11</number>
       </property>
       <property name="value">
        <number>11</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="colorGroupeBox">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>800</y>
      <width>511</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Color</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>495</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="colorGridLayout">
      <item row="0" column="2">
       <widget class="QRadioButton" name="colorRadio2">
        <property name="text">
         <string>Color map</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="colorRadio3">
        <property name="text">
         <string>Cel-shading</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="colorRadio0">
        <property name="text">
         <string>Plain color</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="colorRadio1">
        <property name="text">
         <string>Watercolor</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="colorButton0">
        <property name="text">
         <string>Chose color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="colorButton1">
        <property name="text">
         <string>Chose color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="colorButton2">
        <property name="text">
         <string>Load texture</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="colorButton3">
        <property name="text">
         <string>Load texture</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="shadowGroupBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>800</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Worn shadows</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>341</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="shadowVerticalLayout">
      <item>
       <widget class="QCheckBox" name="shadowCheckBoxLightDir">
        <property name="text">
         <string>Directs the light</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="shadowCheckBoxMorpho">
        <property name="text">
         <string>Do the morphology filter</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="lightShadowPosLabel">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Vertical position of the light :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lightShadowPosValue">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;45&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="lightShadowSlider">
          <property name="maximum">
           <number>89</number>
          </property>
          <property name="value">
           <number>45</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="shadingGroupeBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>800</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Shading</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>341</width>
       <height>113</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCheckBox" name="shadeCheckBoxLightDir">
        <property name="text">
         <string>Directs the light</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="shadeRadio0">
          <property name="text">
           <string>Blur</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="shadeRadio1">
          <property name="text">
           <string>Details</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="shadeRadio2">
          <property name="text">
           <string>Overlay</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="shadeRadio3">
          <property name="text">
           <string>Wartercolor</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lightShadingPosLabel">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Vertical position of the light :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lightShadingPosValue">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;45&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="lightShadingSlider">
          <property name="maximum">
           <number>89</number>
          </property>
          <property name="value">
           <number>45</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QCheckBox" name="doDefaultShading">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>910</y>
      <width>291</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Do the default shading</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1586</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="loadModelAction"/>
    <addaction name="generateModelAction"/>
    <addaction name="screenshotAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="helpAction"/>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="loadModelAction">
   <property name="text">
    <string>Load DEM</string>
   </property>
   <property name="statusTip">
    <string>Load a DEM (.asc)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="screenshotAction">
   <property name="text">
    <string>Screenshot</string>
   </property>
   <property name="statusTip">
    <string>Save a screenshot of the openGL widget in a file (.png)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="statusTip">
    <string>Close the app</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionMesh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mesh</string>
   </property>
  </action>
  <action name="actionHeight_Map">
   <property name="text">
    <string>Height Map</string>
   </property>
  </action>
  <action name="actionNormal_Map">
   <property name="text">
    <string>Normal Map</string>
   </property>
  </action>
  <action name="actionCurvature_Map">
   <property name="text">
    <string>Curvature Map</string>
   </property>
  </action>
  <action name="actionLight_Map">
   <property name="text">
    <string>Light Map</string>
   </property>
  </action>
  <action name="generateModelAction">
   <property name="text">
    <string>Generate aheight map</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lightShadingSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lightShadingPosValue</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>616</x>
     <y>884</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>862</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lightShadowSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lightShadowPosValue</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>935</y>
    </hint>
    <hint type="destinationlabel">
     <x>556</x>
     <y>913</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
